lib/knapsack_pro/config/ci/buildkite.rb

```diff
@@ -33,6 +33,10 @@ def project_dir
         def user_seat
           ENV['BUILDKITE_BUILD_AUTHOR'] || ENV['BUILDKITE_BUILD_CREATOR']
         end
+
+        def detected
+          ENV['BUILDKITE'] == 'true' ? self.class : nil
```

Comment:
```suggestion
          ENV.key?('BUILDKITE') ? self.class : nil
```


This would be enough.  I see sometimes you check if the key is present and another time, you check the value is 'true'. Maybe for the sake of simplicity, just check the key is present. That's enough to detect the CI provider.

Sorry for now making it clear enough. You probably just followed my comment from Trello https://trello.com/c/EXlTjNuu/70-%F0%9F%A7%A9-queue-mode-by-default-auto-enable-knapsackprofixedqueuesplittrue-for-some-of-ci-providers#comment-647e02b39fbe7e8a2c7808f0